buildContainers.nf

```diff
@@ -211,8 +212,8 @@ def helpMessage() {
   log.info "       Default: all"
   log.info "       Possible values:"
   log.info "         all, fastqc, freebayes, gatk, igvtools, multiqc, mutect1"
```

Comment:
Should be mutect2 I guess

Reply:
No, `mutect2` is shipped with the `GATK` container

Reply:
IC, of course. Shall I merge this first, or shall I wait for #567  ?